### ISSUE TITLE
Fix Accessibility issues on Windows

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-multiplatform#1344]
1.0.0-beta5

1. Install [NVDA](https://www.nvaccess.org/)
2. open run1 example in Compose sources

Bugs:
1. If custom density is set - NVDA speaks the wrong text
2. If we hover Text in TextField from bottom to up - NVDA doesn't speak the text
3. If we hover some components - NVDA speaks the text behind the application Window (in Slack, for example)

**Comment by akurasov** [comment]
priority tag removed for tracking purposes

**Comment by VeselovAlex** [comment]
@igordmn Do the issues in description exist when display has default (100%) scale factor?

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5201">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
